paperslicer/utils/exports.py

```
@@ -0,0 +1,134 @@
+    ]
+    count = 0
+    with open(out_path, 'w', encoding='utf-8') as out:
+        for name in sorted(os.listdir(meta_dir)):
+            if not name.endswith('.json'):
+                continue
+            path = os.path.join(meta_dir, name)
+            try:
+                with open(path, 'r', encoding='utf-8') as f:
+                    d = json.load(f)
+            except Exception:
+                continue
```

Ruff (0.12.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)